src/Mentat.UI/Controllers/MentorController.cs

```diff
@@ -10,21 +13,41 @@ public IActionResult Index()
             return View();
         }
 
-        public IActionResult AddAssignment()
+
+        public IActionResult AddAssignment(Models.Assignment assignment)
         {
+ 
             return View();
         }
 
         [HttpPost]
         [ValidateAntiForgeryToken]
         public ActionResult SubmitForm(Models.Assignment assignment)
         {
+
+            //DEMO OF HOW VALUES ARE STORED
             var mentorName = assignment.MentorName;
             var assignmentName = assignment.AssignmentName;
             var assignmentType = assignment.AssignmentType;
             var sampleExecutableName = assignment.SampleExecutableName;
             var testFileNames = assignment.TestFileNames;
+
+            // If form is correct then moves on to the next view 
+            // otherwise view is reloaded and user must retry filling the form
+            if (ModelState.IsValid)
+            {
+                return RedirectToAction("SendData", assignment);
+            }
+ 
             return View();
         }
+
+        // On Form Validation Success Go To This View
+        public IActionResult SendData(Models.Assignment assignment)
+        {
+            return View("SubmitForm");
+        }
```

Comment:
Remove this method. This is paired with previous comment.

Reply:
This action has now been replaced with "Success"  for the much more desired  ./Success/ URL.